packages/client-sdk-web/src/index.ts

```diff
@@ -107,6 +113,9 @@ export {
   CredentialProvider,
   StringMomentoTokenProvider,
   EnvMomentoTokenProvider,
+  AllDataReadWrite,
+  InternalSuperUserPermissions,
```

Comment:
This one is private, we don't want to export it from the main index.  You could put another file at `src/internal/utils/auth.ts` and export it from there, and then when you import it from the console you will need to import it from `dist/src/internal/utils/auth`.  But we don't want to advertise this one in the main index.